Erstes_Konzept/Erstes_Konzept.tex

```diff
@@ -7,6 +7,36 @@ \section{Wirkung}
 \section{Zielgruppe}
 
 \chapter{Ideenbeschreibung}
+Momentan fährt der Roboter nur auf eine Position mit dem Behälter darauf. Diese fahrt wird durch die Software des Logistiksystems mitsamt des Roboters gesteuert. Neu soll die Software dieses Herstellers so erweitert werden (Siehe \ref{sec:roboterSWAnpassung}), dass der Roboter einen vordefinierten Pfad abfährt mit einem Spezifischen RFID Lesebehälter/Suchbehälter (Siehe \ref{sec:behaelterMitRFID}) und dabei noch Aktionen ausführt, wie einen Behälter für Kurze Zeit auszutauschen.
+Würde nun ein Buch, welches von einer Bibliothek oder einem anderen Kunden bestellt wurde, nicht im Entsprechenden Behälter zu finden sein, soll neu der Spezielle RFID Lese-/Suchbehälter in das Hochregallager geschickt werden.
+
+
+Dabei gibt es 2 Unterschiedliche Suchvorkehrungen. Als erstes würde der Roboter mit dem Lese-/Suchbehälter in alle Gassen geschickt, wo Roboter durch jede Reihe nacheinander fährt. Dabei wird jeweils vom Lese-/Suchbehälter der Vordere Behälter nach dem fehlenden Exemplar abgesucht. So kann in einer Kurzer Zeit ca. 50\% des Lagers nach dem deplatzierten Exemplar abgesucht werden. Würde nach diesem Suchvorgang die Lokalisation des Exemplares nicht erfolgreich abgeschlossen werden, würde in der Nacht die 2 Suchfunktion starten, bei welcher der Lese-/Suchbehälter im Hochregallager in der Höhe jeden dritten Platz eines äusseren bereits abgesuchten Behälters für eine kurze Zeit tauscht. Während der Lese-/Suchbehälter am äusseren Platzt ist sucht er im hinteren Behälter nach dem Exemplar sowie in den zu dem abgesuchten behälter direkt darüber sowie darunter befindlichen Behälter.
```

Comment:
... würde in der Nacht die **zweite** Suchfunktion starten, ...
Zahlen bis 13 in Texten ausschreiben:
Zwei Augenpaare sind besser als eines.
Sechs Banditen zu seiner Rechten und ein Kamerad zu seiner Linken ...
Aber ein KB hat nach Speicherhersteller 1000 Bytes, während die korrekte Bezeichnung KiB die intuitiv richtige Anzahl (nämlich 1024 Bytes) bezeichnet.

Anm. Siehe Kommentar oben 😅